hls4ml/writer/vivado_accelerator_writer.py

```diff
@@ -188,6 +216,35 @@ def write_axi_wrapper(self, model):
         f.close()
         fout.close()
 
+    def modify_project_cpp(self, model):
```

Comment:
Lol. Just add a flag to not write the interface in the VivadoWriter instead of this approach.